src/main/java/edu/cornell/kfs/concur/batch/service/impl/ConcurRequestExtractFileValidationServiceImpl.java

```diff
@@ -184,22 +182,29 @@ private boolean employeeIdIsValid(ConcurRequestExtractRequestDetailFileLine deta
             return false;
         }
         else {
-            try {
-                Person employee = getPersonService().getPersonByEmployeeId(detailFileLine.getEmployeeId());
-                if (ObjectUtils.isNotNull(employee)) {
-                    return true;
-                }
-                else {
-                    detailFileLine.getValidationResult().addMessage(getConfigurationService().getPropertyValueAsString(ConcurKeyConstants.CONCUR_REQUEST_EXTRACT_EMPLOYEE_ID_NOT_FOUND_IN_KFS));
-                    return false;
-                }
-            }
-            catch (Exception e) {
-                detailFileLine.getValidationResult().addMessage(getConfigurationService().getPropertyValueAsString(ConcurKeyConstants.CONCUR_REQUEST_EXTRACT_EMPLOYEE_ID_NOT_FOUND_IN_KFS));
+            boolean validPerson = getConcurPersonValidationService().validPerson(detailFileLine.getEmployeeId());
+            if(validPerson) {
+                return true;
+            } else {
+                detailFileLine.getValidationResult().addMessage(getConfigurationService().getPropertyValueAsString(ConcurKeyConstants.CONCUR_EMPLOYEE_ID_NOT_FOUND_IN_KFS));
                 return false;
             }
         }
     }
+    private boolean validateAddressIfCheckPayment(ConcurRequestExtractRequestDetailFileLine detailFileLine) {
```

Comment:
This method is present in 2 services with slight variations. I think it might be better to extract the code that validates the address if check payment into the new validation service. Then you can call that here and just add the appropriate error message.